src/utils/types.ts

```diff
@@ -60,3 +60,10 @@ export interface UpdatedNews {
 	source: string;
 	news: News;
 }
+
+export interface Config {
+	webhookUrl: string;
+	roleID: string;
+	sources: string[];
+	cacheFile: string;
```

Comment:
is it necessary to allow customizing the cache file path? esp since its hardcoded in `.gitignore` and `prettierignore`